backend/download/tests/test_full_data.py

```diff
@@ -24,7 +24,7 @@ def test_timeline_full_data(small_mock_corpus, index_small_mock_corpus, small_mo
 
     group = tasks.term_frequency_full_data_tasks(full_data_parameters, 'date_term_frequency')
     results = group.apply().get()
-    filename = tasks.make_term_frequency_csv(results, full_data_parameters)
+    filename = tasks.make_term_frequency_csv(results, full_data_parameters, 0)
```

Comment:
It would be easier to parse the parameters in the tests if `log_id` was explicitly named here.